seekers/tests.py

```diff
@@ -1,6 +1,7 @@
 from apartments.models import City
 from .forms import SeekerCreationForm
 from django.urls import reverse
+from users.tests import user_model  # noqa: F401
```

Comment:
You should not import fixtures but put them inside a conftest file. with that, they'll be automatically available for use across the scope they're in.

read more:
https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Don't ignore the linting issues, they're there for a reason.

Reply:
Yes, I saw that. but I think that moving all the fixtures to a new file and changing all the tests in the process is not in the scope of this PR. Can I open an issue for the redesigning of the fixtures and the tests and have this PR merged?

Reply:
As I mentioned in the Additional Information part:
Found a bug/issue with flake8, when i tried to import a fixture and use it flake8 was confused,
We will probably have to redisign the fixtures in all the tests files, for now I added # noqa: F811 and # noqa: F401 to
make flake8 ignore those false positives...

Reply:
No, what you should do in that case is first to open a PR that fixes this issue, and then rebase this PR on top of it.

Reply:
Ok, I'll start working on it
👍

Reply:
@Omeramsc I pushed a new PR that moves everything to conftest.py